### DISCUSSION TITLE
Calling reusable workflow and seeing its run history

### DISCUSSION THREAD

**Discussion by chipzoller** [community/community#75365] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi there, been searching and playing with this for a while without finding a solution. When reusing one workflow in another, how can you get the run history of the called workflow show up in the UI? The caller is able to call the reused workflow just fine, but I'm wanting to see that reused workflow have its history show up under itself under the list of actions. Is there a way to achieve this?

**Comment by b8zeek** [comment, marked as answer]
Hey Chip. 👋🏼 

No, there is no way to achieve what you want as the reused workflow does not actually run as a separate workflow so its run history is not updated. Maybe GitHub updates this in the future but currently, there is no way to see the run history because when you use the `uses` keyword to reuse a workflow, the steps of the reused workflow are essentially copied into the calling workflow.

Any time, you can request the feature and GitHub might add it if they find it valuable:

https://support.github.com/contact/feedback

Best of luck bro! 🍺 

🍀

**Reply by chipzoller** [reply]
Thank you for answering! This is what I expected.

**Reply by b8zeek** [reply]
I'm glad I helped! Take care Chip. ☺️

🍀
